src/solver/conv_ocl_dir2Dfwd_exhaustive_search.cpp

```diff
@@ -293,10 +293,12 @@ ConvOclDirectFwdLegacyExhaustiveSearch::SearchImpl(const ConvolutionContext& ctx
     auto bias_ocl_ptr = bias_ocl_buf.get();
 #else
     auto& profile_h           = ctx.GetStream();
-    const auto& invoke_params = invoke_ctx.CastTo<conv::DataInvokeParams>();
-    const auto bot_ocl_ptr    = invoke_params.tensors.in;
-    const auto top_ocl_ptr    = invoke_params.tensors.out;
-    const auto wei_ocl_ptr    = invoke_params.tensors.w;
+    const auto& invoke_params = invoke_ctx.CastTo<miopen::fusion::FusionInvokeParams>();
+    const auto bot_ocl_ptr    = invoke_params.in;
+    const auto top_ocl_ptr    = invoke_params.out;
+    const auto wei_ocl_ptr =
+        dynamic_cast<miopen::fusion::ConvolutionOpInvokeParam&>(*invoke_params.op_args.params[0])
+            .weights;
```

Comment:
- Revert this change
- In `ConvOclDirectFwdFused::Search()`, construct `DataInvokeParams` instance from `FusionInvokeParams` instance. Pass the former to `ConvOclDirectFwd{}.Search()`.